apps/custom-config/wakeup-effect.js

```diff
@@ -19,6 +19,7 @@ var AWAKE_EFFECT = 'rokid.custom_config.wakeup_sound'
 
 var WAKE_SOUND_OPEN_DEFAULT = '唤醒应答已设置为默认声音'
 var WAKE_SOUND_OPEN_CUSTOM = '唤醒应答已设置<phoneme alphabet="py" ph="wei2">为</phoneme>你录制的声音'
+var WAKE_SOUND_OPEN = '已为你开启'
```

Comment:
Is this incomplete?

Reply:
It's the tts for intent(designed in v7.28).